extensions/pubsub/standard/sections/clause_9_pubsub-message-payload.adoc

```diff
@@ -29,7 +29,8 @@ include::../recommendations/pubsub-message-payload/REC_crs.adoc[]
 ==== Identifier
 
 A universally unique identifier of the message using the UUID standard (RFC4122). The identifier is generated by the originator of the message.
-It remains the same throughout the lifetime of the message.
+
+It remains the same throughout the lifetime of the message.  Note that multiple messages may communicate updates on the same resource.  The identifier is also valuable for "replay" feed services, where a Pub/Sub client may have gone offline for a period of time.
```

Comment:
> Note that multiple messages may communicate updates on the same resource. 

What's the consequence of this? Is there any? Any recommended behaviour?

Reply:
> > Note that multiple messages may communicate updates on the same resource.
> 
> What's the consequence of this? Is there any? Any recommended behaviour?

I've added a recommendation for `properties.resource_id` in this context.